dark/reads.py

```diff
@@ -133,6 +135,26 @@ def walkHSP(self, hsp):
             readOffset += 1
             subjectOffset += 1
 
+    def checkAlphabeth(self, count=10):
+        """
+        A function which checks whether the sequence in a L{dark.Read} object
+        corresponds to its readClass. For AA reads, more testing is done in
+        dark.Read.AARead.checkAlphabeth.
+
+        @param count: A C{int} of how many amino acids should be considered.
+        @return: C{True} if the sequence corresponds from its readClass or
+            C{False} if the sequence doesn't correspond.
+        @raise ValueError: If the sequence and the readClass don't match, raise
+            a ValueError.
+        """
+        readLetters = set(self.sequence.upper()[:count])
```

Comment:
So if you say
```seq[:None]``` you get the whole sequence. So I can just change the default of `count` to `None`.